python/cudf/cudf/tests/test_json.py

```diff
@@ -660,3 +664,49 @@ def test_json_types_data():
         pdf, schema=df.to_arrow().schema, safe=False
     )
     assert df.to_arrow().equals(pa_table_pdf)
+
+
```

Comment:
Is there a test where we pass `dtype`, with nested types?

Reply:
Done 👍